usagov/vote-gov-drupal
# VOTE-2153 update accessibility focus outline

## Jira ticket

[VOTE-2153](https://cm-jira.usa.gov/browse/VOTE-2153)

## Description

1. Add offset for focus outline for links
2. Update focus outline color for homepage hero

## Deployment and testing

### Post-deploy steps

1. run `npm run build` in votegov theme directory
2. run `lando drush cr`

### QA/Testing instructions

1. visit http://vote-gov.lndo.site/
2. focus on hero button and check the color is overridden
3. test in high contrast mode and confirm it revert backs to the default outline focus color
4. Scroll down to the USAgov election links and confirm that the focus outline is offset

## Checklist for the Developer




## Checklist for the Peer Reviewers
